### PR TITLE
T1977

### PR DESCRIPTION
**Pull request by mjschranz**
https://webmademovies.lighthouseapp.com/projects/65733/tickets/1977-fix-styling-in-image-plugin-editor

### CODE REVIEW

**Comment by cadecairos** [comment]
templates/assets/editors/image/popcorn.image.editor.js :
    Unused Variables:
        _disabledClass(10),

**Comment by cadecairos** [comment]
HTML5 Lint Issues for file: templates/assets/editors/image/popcorn.image.editor.html

Error: The “label” element may contain at most one “input”, “button”, “select”, “textarea”, or “keygen” descendant.
From line 27, column 17; to line 27, column 76

Warning: “label” element with multiple labelable descendants.
From line 24, column 15; to line 24, column 47

Error: The element “label” must not appear as a descendant of the “label” element.
From line 35, column 17; to line 35, column 23

Error: The “label” element may contain at most one “input”, “button”, “select”, “textarea”, or “keygen” descendant.
From line 36, column 17; to line 36, column 111

Warning: “label” element with multiple labelable descendants.
From line 32, column 15; to line 32, column 47

Error: The element “label” must not appear as a descendant of the “label” element.
From line 37, column 17; to line 37, column 23

Error: The “label” element may contain at most one “input”, “button”, “select”, “textarea”, or “keygen” descendant.
From line 38, column 17; to line 38, column 116

Warning: “label” element with multiple labelable descendants.
From line 32, column 15; to line 32, column 47

Error: Element “div” not allowed as child of element “label” in this context. (Suppressing further errors from this subtree.)
From line 39, column 17; to line 39, column 50

Error: The element “label” must not appear as a descendant of the “label” element.
From line 40, column 19; to line 40, column 25

Error: The “label” element may contain at most one “input”, “button”, “select”, “textarea”, or “keygen” descendant.
From line 41, column 19; to line 41, column 119

Warning: “label” element with multiple labelable descendants.
From line 32, column 15; to line 32, column 47

Error: End tag “div” seen, but there were open elements.
From line 43, column 13; to line 43, column 18

Error: Unclosed element “label”.
From line 32, column 15; to line 32, column 47

File: templates/assets/editors/image/popcorn.image.editor.html [hunk 1]
```diff
@@ -2,78 +2,49 @@
...
+            </div>
+
+            <div class="editor-radio-wrapper">
+              <label class="butter-form-radio">
+                <input type="radio" name="source-type" id="editor-flickr-radio"  class="editor-input-radio" >
+                Flickr Slideshow
+                <label>Tag Search</label>
+                <input type="text" id="editor-flickr-tag" class="editor-text-input" data-manifest-key="tags" />
+                <label>Gallery URL</label>
+                <input type="url" id="editor-flickr-url" class="editor-text-input" data-manifest-key="photosetId" />
```
Comment (cadecairos) [inline templates/assets/editors/image/popcorn.image.editor.html]: This should not be a url input.

**Comment by cadecairos** [comment]
Number defaults to 0, should be something sane, like 5.

When I change the transition drop down, it always changes back to pop-up

**Comment by cadecairos** [comment]
Changing the start and end times in the editor doesn't update the track event.

**Comment by cadecairos** [comment]
selecting URL and clicking on the text box will immediately force focus away from the textbox.
edit: happens with all textboxes, focus is given to the checkbox. I believe it's related to invalid HTML.

I don't think the user should be able to manually select the first option. it should be programmiatically enabled when an image is dropped.

**Comment by cadecairos** [comment]
### Linting JS files

templates/assets/editors/image/popcorn.image.editor.js: line 186, col 8, Missing semicolon.

1 error

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/2c828b1b49dafdf/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @jbuck received. Current queue size: 2

Live output at: http://botio.butyr.org:8000/3423a13f137da6a/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/2c828b1b49dafdf/output.txt

Total script time: 1.25 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/3423a13f137da6a/output.txt

Total script time: 1.37 mins

Server is running on http://pr809.butyr.org/templates/basic/
Run the Test Suite: http://pr809.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/126aa9196790b71/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/eb8304a84917633/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/126aa9196790b71/output.txt

Total script time: 1.02 mins

Server is running on http://pr809.butyr.org/templates/basic/
Run the Test Suite: http://pr809.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/eb8304a84917633/output.txt

Total script time: 1.34 mins

**Pull request merged**